test/optimus/pr/ValidateOptimus.wdl

```diff
@@ -51,6 +52,14 @@ task ValidateOptimus {
       fail=true
     fi
 
+    for htmlfile in ${sep=' ' fastqc_htmls}; do
```

Comment:
I guess you can't easily check the zips here? The test could at least assert that the number of zip files generated is correct.